src/credits.md

```diff
@@ -6,182 +6,223 @@ permalink: /credits/
+      Kyoto
+    </p>
+    <h3>Website Development and Design</h3>
+    <p>
+      Center for Interdisciplinary Digital Research (CIDR), Stanford University
+      Library:<br />
+      <strong>Glen Worthey</strong><br />
+      <strong>Scott Bailey</strong><br />
+      <strong>Peter Broadwell</strong><br />
+      <strong>Javier de la Rosa</strong><br />
+      <strong>Juan Pablo López Franco</strong><br />
+      <strong>Simon Wiles</strong><br />
+      <strong>Vijoy Abraham</strong>
+    </p>
+
+    <h3>Translation of Noh Plays</h3>
+    <p><strong>Rebecca Teele Ogamo</strong>, Kongō School</p>
+
+    <h3>Editor</h3>
+    <p>
+      <strong>Ellen O'Connell Whittet</strong>, University of California, Santa
+      Barbara
+    </p>
+
+    <h3>Additional Collaborators</h3>
+    <p>
+      <strong>NAKAMURA Itsuko</strong>, American Consortium for Japanese
+      Studies<br />
+      <strong>TAKAHASHI Yoko</strong>, Kyoto City University of Arts
+      <br /><strong>David Yu</strong>, Center for Teaching and Learning at the
+      University of the Pacific
+    </p>
+
+    <h3>Performers</h3>
+    <h5>ACADEMIC EXAMPLES</h5>
+    <p>
+      Dancer: <strong>KONGŌ Tatsunori</strong><br />
+      Singers: <strong>UDAKA Tatsushige</strong>,
+      <strong>UDAKA Norishige</strong>, <strong>ARIMATSU Ryōichi</strong><br />
+      Drum stick: <strong>NAKATA Hiromi</strong><br />
+      Shoulder drum: <strong>HAYASHI Yamato</strong><br />
+      Hip drum: <strong>TANIGUCHI Masatoshi</strong><br />
+      Flute: <strong>MORITA Yasuyoshi</strong>
+    </p>
+
+    <h5>Hashitomi</h5>
+    <p>
+      Shite: <strong>KONGŌ Tatsunori</strong><br />
+      Waki: <strong>ARIMATSU Ryōichi</strong><br />
+      Kyōgen: <strong>SHIGEYAMA Shime</strong><br />
+      Shoulder drum: <strong>NARITA Tatsushi</strong><br />
+      Hip drum: <strong>TANIGUCHI Masatoshi</strong><br />
+      Flute: <strong>MORITA Yasuyoshi</strong><br />
+      Jiutai: <strong>UDAKA Michishige</strong>, Leader<br />
+      <strong>MUKAI Hiroki</strong><br />
+      <strong>SHIGEMOTO Masaya</strong><br />
+      <strong>TANAKA Toshifumi</strong><br />
+      <strong>TANIGUCHI Masahiko</strong><br />
+      <strong>TESHIMA Kouji</strong><br />
+      <strong>TESHIMA Yukihiro</strong><br />
+      <strong>YAMADA Isumi</strong><br />
+      Kōken: <strong>HIROTA Yukitoshi</strong><br />
+      Stage assistant: <strong>SOUMYOU Tadasuke</strong>
+    </p>
+
+    <h5>Kokaji</h5>
+    <p>
+      Shite: <strong>UDAKA Tatsushige</strong><br />
+      Waki: <strong>ARIMATSU Ryōichi</strong><br />
+      Wakizure: <strong>HARA Riku</strong><br />
+      Kyōgen: <strong>SHIGEYAMA Shime</strong><br />
+      Drum stick: <strong>MAEKAWA Mitsunaga</strong><br />
+      Shoulder drum: <strong>NARITA Tatsushi</strong><br />
+      Hip drum: <strong>TANIGUCHI Masatoshi</strong><br />
+      Flute: <strong>MORITA Yasuyoshi</strong><br />
+      Jiutai: <br />
```

Comment:
The format here is a little different than for Hashitomi above -- probably this is unintentional?